packages/lisk-transaction-pool/src/transaction_pool.ts

```diff
@@ -43,15 +50,20 @@ interface Queues {
 export class TransactionPool {
 	// tslint:disable-next-line variable-name
 	private readonly _queues: Queues;
+	private readonly EXPIRE_TRANSACTIONS_JOB: number;
```

Comment:
since this is not constant, it should be camelCase too

Reply:
this is the interval of the expireTransaction job. Which will be passed to the constructor, shouldn't it be a constant?

Reply:
if we pass from constructor, i think it's not a constant =p